scene/gui/tree.cpp

```diff
@@ -3834,6 +3847,10 @@ bool Tree::is_editing() {
 	return popup_editor->is_visible();
 }
 
+void Tree::set_editor_selection(int start, int end) {
+	text_editor->select(start, end);
```

Comment:
`text_editor` is now multiline, single-line editor is now under `line_editor`. This function needs to be updated.
Since single-line/multi-line depends on cell mode, maybe selection rows could be optional arguments.